tests/test_loss_mitigation.py

```diff
@@ -70,3 +75,64 @@ def test_photon_loss_mitigation(result_type):
     for keys, value in mitigated_dist.items():
         assert sum(keys) == ideal_photon_count
         assert value > 1e-6
+
+def _compute_random_circ_probs(source_emission, num_photons):
+
+    mzi = (BS() // (0, PS(phi=Parameter("φ_a")))
+           // BS() // (1, PS(phi=Parameter("φ_b"))))
+
+    random_loc = Circuit.decomposition(Matrix.random_unitary(20), mzi,
+                                              phase_shifter_fn=PS,
+                                              shape="triangle")
+    # Processor config
+    processor = Processor("SLOS", random_loc)
+    processor.min_detected_photons_filter(0)
+    processor.thresholded_output(True)
+
+    # Source properties
+    src = Source(emission_probability=source_emission)
+    processor.source = src
```

Comment:
Please use `NoiseModel` instead of `Source` as it's deprecated.

A way of doing this is when you instantiate the Processor:
```python
processor = Processor("SLOS", random_loc, noise=NoiseModel(transmittance=source_emission))
```

Reply:
Sorry, my bad. I forgot. I took this and updated the existing test to use `NoiseModel()` too.